bears/general/TextLintBear.py

```diff
@@ -0,0 +1,194 @@
+                    NpmRequirement('textlint-plugin-html', '0.1.5'),
+                    NpmRequirement('textlint-plugin-rst', '0.1.1'),
+                    NpmRequirement('textlint-rule-alex', '1.2.0'),
+                    NpmRequirement('textlint-rule-common-misspellings',
+                                   '1.0.1'),
+                    NpmRequirement('textlint-rule-date-weekday-mismatch',
+                                   '1.0.5'),
+                    NpmRequirement('textlint-rule-ginger', '2.1.0'),
+                    NpmRequirement('textlint-rule-max-comma', '1.0.4'),
+                    NpmRequirement('textlint-rule-max-number-of-lines',
+                                   '1.0.3'),
+                    NpmRequirement('textlint-rule-ng-word', '1.0.0'),
+                    NpmRequirement('textlint-rule-no-dead-link', '3.1.1'),
+                    NpmRequirement('textlint-rule-no-empty-section', '1.1.0'),
+                    NpmRequirement('textlint-rule-no-start-'
+                                   'duplicated-conjunction', '1.1.3'),
+                    NpmRequirement('textlint-rule-no-todo', '2.0.0'),
+                    NpmRequirement('textlint-rule-period-in-list-item',
+                                   '0.2.0'),
+                    NpmRequirement('textlint-rule-rousseau', '1.4.5'),
+                    NpmRequirement('textlint-rule-unexpanded-acronym',
+                                   '1.2.1'),
+                    NpmRequirement('textlint-rule-write-good', '1.6.0'),
+                    PipRequirement('docutils-ast-writer', '0.1.2')}
+    AUTHORS = {'The coala developers'}
+    AUTHORS_EMAILS = {'coala-devel@googlegroups.com'}
+    LICENSE = 'AGPL-3.0'
+    CAN_DETECT = {'Formatting', 'Grammar', 'Spelling'}
+    SEE_MORE = 'https://github.com/textlint/textlint'
+
+    @staticmethod
+    def generate_config(filename, file,
+                        check_todos: bool=None,
+                        dont_start_with_duplicated_conjunction: bool=True,
+                        no_empty_section: bool=True,
+                        check_date_weekday_mismatch: bool=True,
+                        check_grammar: bool=True,
+                        max_lines_per_file: int=300,
+                        max_comma_per_sentence: int=4,
+                        check_no_good_words: typed_list(str)=[],
+                        period_in_list_item: bool=True,
+                        minimum_acronym_length: int=3,
+                        maximum_acronym_length: int=5,
+                        ignore_acronyms: typed_list(str)=[],
+                        check_rousseau: bool=True,
+                        check_offensive_expressions: bool=True,
+                        check_common_misspellings: bool=True,
+                        write_good: bool=True,
+                        check_relative_links: bool=False,
+                        base_uri: str='null',
```

Comment:
what about leaving the default empty? Though it seems that textlint doesn't like this :3